src/test/resources/no/nav/familie/ef/sak/beregning_barnetilsyn_en_mnd.feature

```diff
@@ -0,0 +1,81 @@
+# language: no
+# encoding: UTF-8
+
+Egenskap: Beregning av barnetilsyn (en periode)
+
+  Scenario: Varierende utgifter og inntekter, satsperiode og antall barn for 2022
```

Comment:
Kunne det vært mulig å dele opp dette? Slik at man tydeligere ser hvordan utgifter, inntekter, satsperiode og barn påvirker periodebeløp. For eksempel en test for antall barn, en for inntekter, også beholde denne for kombinasjoner som ikke blir dekt av de isolerte testene

Reply:
Ja vi hadde veldig lyst til å skrive flere cucumbertester, men satte egentlig strek for å komme videre. Mye av det du nevner er dekket av unit-tester. Vi la inn en cucumber-kaosvariant her som ble veldig vanskelig å lese i unit-test. 

Kan gjerne legge inn noen flere cucumbertester, men kan kanskje vente til vi har mer validering på plass?

Reply:
Men kan godt dele opp denne ja :)